users/api/http/transport.go

```diff
@@ -57,13 +60,34 @@ func MakeHandler(svc users.Service, tracer opentracing.Tracer, l log.Logger) htt
 		opts...,
 	))
 
+	mux.Post("/passwd/res-req", kithttp.NewServer(
```

Comment:
There is a reason to don't use `password` here? I would use `password/request` here

Reply:
Also, this is a request for password change, so I recommend that you use `Patch` instead of post. So based on the REST standards, think about request like this one:
```
PATCH /user
{
    "password": "new_pass"
}
```

Reply:
this res-req endpoint initiates the flow for password password/reset is actually where the password is changed

Reply:
@mteodor, please use `PATCH /user`, as @anovakovic01 suggested

Reply:
@mteodor I understood that it initiates. That's why `password/request` makes sense.